### ISSUE TITLE
Spark vs Particle CLI tools, and npm issues

### ISSUE THREAD

**Issue by QCaudron** [spark/particle-cli#249]
A couple of things :

- It's not clear to me, as a user, whether we should be using spark-cli to deal with Cores, and particle-cli to deal with Photos, or if particle-cli can handle both. The spark-cli repo is deprecated and suggests we use particle-cli instead; however, the vast number of guides and tutorials out there are still pointing to spark-cli. As such, I think this should be make obvious in the docs and here on Github : are we to completely forget spark-cli, even if we're trying to claim Cores ?
- `npm install -g particle-cli` requires sudo rights. Do we necessarily want the CLI tools installed globally, or can we be happy with them installed in the user's local directory, and drop the `-g` that's causing some users confusing when installing ? I'm in the process of helping a user who doesn't have sudo access on her Mac, and the global flag is hence causing the issue to fail. The workaround is to install without `-g` and put the path to `particle.js` into the PATH variable, which can be easily scripted without needing sudo access.

TL; DR : **can particle-cli be used to claim Cores ?** Let's make it clear somewhere, and perhaps have a note in the docs on installing without sudo access.

**Comment by monkbroc** [comment]
Thanks for those comments.

Yes, particle-cli works with Cores. spark-cli is no longer needed.

There are many variations of installing with npm. For example I configured my npm to install global packages in `~/.npm-global` by setting `prefix=~/.npm-global` in `~/.npmrc`. It's simpler to install globally with sudo, but alternatives are possible if that's not an option.

In any case the installation of the CLI is an area that is being improved right now because it is definitely a barrier for new users.

**Comment by brycekahle** [comment]
Also see https://docs.npmjs.com/getting-started/fixing-npm-permissions

**Comment by QCaudron** [comment]
Thanks, @monkbroc and @brycekahle. Thoughts on whether this is worth making clearer in the docs ?

**Comment by monkbroc** [comment]
I clarified the note in the spark-cli repository that mentions it is deprecated.
